packages/deployment/lib/kes.js

```diff
@@ -287,6 +295,27 @@ class UpdatedKes extends Kes {
   }
 
 
+  /**
+   * Updates lambda/sqs configuration to include an sqs dead letter queue
+   * matching the lambdas's name (e.g. {lambda.name}DeadLetterQueue)
+   * @returns {void} Returns nothing.
+   */
+  addLambdaDeadLetterQueues() {
+    const lambdas = this.config.lambdas;
+    Object.keys(lambdas).forEach((key) => {
+      const lambda = lambdas[key];
+      if (lambda.namedLambdaDeadLetterQueue) {
+        console.log(`Adding named dead letter queue for ${lambda.name}`);
+        const queueName = `${lambda.name}DeadLetterQueue`;
+        this.config.sqs[queueName] = {
+          MessageRetantionPeriod: this.config.DLQDefaultMessageRetentionPeriod,
```

Comment:
I was thinking, should we have a default here so they don't have to set it if they don't want to?

Reply:
@laurenfrederick  I'm not sure what you mean on this comment -- as this is a the doc PR, I'll see about chatting offline about this and update the ticket requirements if needed for close.